### ISSUE TITLE
Segment page event missing properties

### ISSUE THREAD

**Issue by klintan** [DavidWells/analytics#167]
Thanks for a great lib! 

Using this for segment, however it seems unfortunately the "name" root property in the page event is missing, meaning passing this on to Amplitude for instance will fail.

```
{
  "anonymousId": "xxxxx",
  "category": null,
  "context": {
    "ip": "73.71.41.238",
    "library": {
      "name": "analytics.js",
      "version": "4.0.8"
    },
    "locale": "en-US",
    "page": {
      "path": "/users",
      "referrer": "",
      "search": "",
      "title": "xxxx",
      "url": "http://xxxx.com/users"
    },
    "userAgent": "xxxxx"
  },
  "integrations": {},
  "messageId": "ajs-c9d5e3112157174c42361c837fc2422f",
  "name": null,
  "originalTimestamp": "2021-04-11T00:11:49.734Z",
  "properties": {
    "hash": "",
    "height": 1259,
    "path": "/users",
    "referrer": "",
    "search": "",
    "title": "xxxx",
    "url": "http://xxxx.com/users",
    "width": 634
  },
  "receivedAt": "2021-04-11T00:11:49.829Z",
  "sentAt": "2021-04-11T00:11:49.736Z",
  "timestamp": "2021-04-11T00:11:49.827Z",
  "type": "page",
  "userId": "xxx"
}
```

As you can see the "name" property is the culprit. You can extend the properties with a name of course, but that will always be contained within the properties section. 
I looked into the code for putting in a PR to fix this, but my React Typescript knowledge might be lacking a bit because it looked as if it would be a bit more extensive fix than just adding it to the segment io plugin module.

Changing the name to something other than `null` fixes it.

Perhaps you can shed some light on this ? Maybe i'm just missing something trivial.

**Comment by DavidWells** [comment]
Page sends through this data by default https://github.com/DavidWells/analytics/blob/1d0aeb1ff0a751d3ee03e6a2806c52c00f6552b3/packages/analytics-core/src/modules/page.js#L60-L68 to here https://github.com/DavidWells/analytics/blob/1d0aeb1ff0a751d3ee03e6a2806c52c00f6552b3/packages/analytics-plugin-segment/src/browser.js#L124

It does appear that the first param is the page title. https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/#page

```
const title = 'Pricing'
const otherTitle = 'Segment Pricing'
// Why 2 segment? 😅

analytics.page(title, {
  title: otherTitle,
  url: 'https://segment.com/pricing',
  path: '/pricing',
  referrer: 'https://segment.com/warehouses'
});
```

I'm not sure what the "name" value is used for.

Will submit a fix to update shortly.

**Comment by DavidWells** [comment]
Released via @analytics/segment@1.0.0

By default, the page name is the `document.title` value. 

To have shorter names call page like so:

```js
analytics.page({
  name: 'HomePage'
})
```

**Comment by klintan** [comment]
Awesome thanks!
